plugins/intel-gsc/igsc.quirk

```diff
@@ -1,123 +1,128 @@
-[87d90ca5-3495-4559-8105-3fbfa37b8b79]
+[PCI\DRIVER_xe]
 Plugin = igsc
```

Comment:
is this going to be too much?  Won't it match APUs?

Reply:
I think that's actually a good thing -- no? If the APU exports a `fw_status` then `->probe `will succeed; otherwise we'll get `NOT_SUPPORTED`.

Reply:
Ah I see...  OK.